### PR TITLE
Handle extended chars

### PR DESCRIPTION
**Pull request by rmisio**
#329

Not sure I 100% understand why this works, but the updates here do appear to solve the server failing on extended chars (e.g. entering a ® in the description field of a contract).

Anyhow, despite all the `decode("utf8")` statements (including on the aforementioned `description` field), the server was still throwing a UnicodeEncodeError exception on the saving of a contract with extended chars:

screen shot 2016-05-12 at 11 35 55 am

Once I removed all the `decode("utf8")` statements, the save went through, albeit with the following warning:

screen shot 2016-05-12 at 11 33 54 am

Once I set the server to use utf8 as the default encoding (as described [here](http://stackoverflow.com/a/31137935)), the warning went away.

### CODE REVIEW

File: openbazaard.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
 from twisted.python import log, logfile
 from txws import WebSocketFactory
 
+reload(sys)
```
Comment (rmisio) [inline openbazaard.py]: Not sure if this should live here in our entry point, or somewhere else?

**Comment by hoffmabc** [comment]
I feel like this is potentially a brittle solution to change the default encoding. Maybe some python gurus have a better opinion on this.

**Comment by hoffmabc** [comment]
Please check the travis fails as well.

**Comment by rmisio** [comment]
I agree it has some potential brittleness to it. Hopefully, this PR could at the very least point a more experienced Python dev in the right direction to having our APIs handle extended chars.

FWIW, apparently Python 3 uses utf8 as the standard, so us explicitly setting it is kind of following suit with the future!

Anyways, Travis is choking on the fact that I'm using the built-in function `reload`, which apparently [is required](http://stackoverflow.com/a/3828742) in order to set the default encoding. I think I would need someone to educate me on a way to use that built-in that would make Travis happy (if there is one).

**Comment by hoffmabc** [comment]
@rmisio can you please resolve the pylint issues in the travis log so we can review?

**Comment by cpacia** [comment]
I think it just needs an override on reload I can do that.

**Pull request closed without merging**